plugins/techdocs/package.json

```diff
@@ -52,6 +52,9 @@
     "@material-ui/icons": "^4.9.1",
     "@material-ui/lab": "4.0.0-alpha.57",
     "@material-ui/styles": "^4.10.0",
+    "@testing-library/react": "^12.1.3",
+    "@backstage/test-utils": "^1.0.1-next.1",
+    "testing-library__dom": "^7.29.4-beta.1",
```

Comment:
unfortunately the pre-commit checks didn't let me have it under devDependencies. Maybe because the test-utils is not a *.test.* file? if you have suggestions please let me know!

Reply:
Hmm... Got it. I should be fine because the `@backstage/test-utils` does the same, right @backstage/maintainers ?
https://github.com/backstage/backstage/blob/master/packages/test-utils/package.json#L45-L47

Reply:
If those facilities are used at runtime, then yeah it has to go in the regular dependencies.

That's an argument for making separate test utils modules - then the test utils module can have a regular dependency on this but the _end consumer_ can have a dev dependency on the test library so these don't get bundled into their release artifacts unnecessarily, bloating them. That's why the test utils exist in the first place, instead of putting those pieces of functionality eg directly in core-plugin-api.

But that's a bigger change so maybe you will want to do that separately, if ever.